modules/openideal_toolbar/js/toolbar.js

```diff
@@ -0,0 +1,22 @@
+(function ($, Drupal) {
+
+    'use strict';
+
+    /**
+     * Openideal Toolbar behavior.
+     *
+     * @type {Drupal~behavior}
+     *
+     * @prop {Drupal~behaviorAttach} attach
+     *   Attach icon for OI toolbar.
+     */
+    Drupal.behaviors.openidealToolbar = {
+      attach: function (context, settings) {
+        $('.toolbar-icon-openideal-toolbar-help').once('openideal_toolbar').each(function () {
+          var logo = settings.openidealToolbar.logo;
+          $(this).css('--toolbar-url', `url(${logo})`)
```

Comment:
Should we check if the logo is not empty? (in case someone removed it in any hook_alter)

Reply:
Added additional condition